docs/tutorials/tutorial-sketches-theta.md

```diff
@@ -0,0 +1,302 @@
+      },
+      "inputFormat": {
+        "type": "csv",
+        "findColumnsFromHeader": true
+      }
+    },
+    "tuningConfig": {
+      "type": "index_parallel",
+      "partitionsSpec": {
+        "type": "hashed"
+      },
+      "forceGuaranteedRollup": true
+    },
+    "dataSchema": {
+      "dataSource": "inline_data",
+      "timestampSpec": {
+        "column": "date",
+        "format": "auto"
+      },
+      "dimensionsSpec": {
+        "dimensions": [
+          "show",
+          "episode"
+        ]
+      },
+      "granularitySpec": {
+        "queryGranularity": "day",
+        "rollup": true,
+        "segmentGranularity": "day"
+      },
+      "metricsSpec": [
+        {
+          "name": "count",
+          "type": "count"
+        },
+        {
+          "type": "thetaSketch",
+          "name": "theta_uid",
+          "fieldName": "uid"
+        }
+      ]
+    }
+  }
+}
+```
+
+Notice the `theta_uid` object in the `metricsSpec` list, which directs Druid to apply the `thetaSketch` aggregator on the `uid` column during ingestion.
+
+Click **Submit** to start the ingestion.
+
+## Query the Theta sketch column
+
+Getting a unique count estimate out of a Theta sketch column involves two steps:
+
+1. merging the Theta sketches in the column by means of an [aggregator function](../querying/sql-aggregations.md#theta-sketch-functions), which in Druid SQL is called `DS_THETA`
+2. getting the estimate out of the merged sketch using [`THETA_SKETCH_ESTIMATE`](../querying/sql-scalar.md#theta-sketch-functions).
+
+Between steps 1 and 2, you can apply set functions. We will come to that in a moment.
+
+### Basic counting 
+
+Let's first see what the data looks like in Druid. Run the following SQL statement in the query editor:
+```sql
+SELECT * FROM ts_tutorial
+```
+
+![View data with SELECT all query](../assets/tutorial-theta-06.png)
+
+The Theta sketch column `theta_uid` appears as a Base64-encoded string; behind it is a bitmap.
+
+A simple query to compute distinct counts of user IDs uses `APPROX_COUNT_DISTINCT_DS_THETA` and groups by the other dimensions:
+```sql
+SELECT __time,
+       "show",
+       "episode",
+       APPROX_COUNT_DISTINCT_DS_THETA(theta_uid) AS users
+FROM   ts_tutorial
+GROUP  BY 1, 2, 3
+```
+
+![Count distinct with Theta sketches](../assets/tutorial-theta-07.png)
+
+In the query above, `APPROX_COUNT_DISTINCT_DS_THETA` is equivalent to calling `DS_THETA` and `THETA_SKETCH_ESIMATE` as follows:
+
+```sql
+SELECT __time,
+       "show", 
+       "episode",
+       THETA_SKETCH_ESTIMATE(DS_THETA(theta_uid)) AS users
+FROM   ts_tutorial
+GROUP  BY 1, 2, 3
+```
+
+That is, `APPROX_COUNT_DISTINCT_DS_THETA` applies the following:
+* `DS_THETA`: Creates a new Theta sketch from the column of Theta sketches
+* `THETA_SKETCH_ESTIMATE`: Calculates the distinct count estimate from the output of `DS_THETA`
+
```

Comment:
I think this query just demonstrates how to do the count distinct. Maybe it could be said that there are more unique watchers on 5/21 compared to days 5/20 and 5/19. If a user removes the `__time` grouping, you might be able to compare numbers.